packs/howl-of-the-wild-bestiary/alicorn.json

```diff
@@ -206,7 +206,7 @@
                 "damage": {},
                 "defense": null,
                 "description": {
-                    "value": "<p>You can communicate telepathically with creatures within 30 feet. Once you establish a connection by communicating with a creature, the communication is two-way. You can communicate only with creatures that share a language with you.</p>\n<hr />\n<p><strong>Heightened (6th)</strong> You can communicate telepathically with creatures using shared mental imagery even if you don't share a language <em>telepathy</em> loses the linguistic trait.</p>"
+                    "value": "<p>You can communicate telepathically with creatures within 30 feet. Once you establish a connection by communicating with a creature, the communication is two-way. You can communicate only with creatures that share a language with you.</p>\n<hr />\n<p><strong>Heightened (6th)</strong> You can communicate telepathically with creatures using shared mental imagery even if you don't share a language; <em>telepathy</em> loses the linguistic trait.</p>"
```

Comment:
Telepathy should have the semicolon to match the source

Reply:
Yes.
I check with both the source book and AoN before committing.

Reply:
My mistake - you’re adding the semicolon, not removing it.